docs/deploying/database.md

```diff
@@ -31,3 +31,88 @@ Migrations are [run automatically](https://github.com/elifesciences/elife-xpub-f
+
+The scripts `dump-database.sh` and `restore-database.sh`, on PATH in the servers, can respectively be used to perform a new dump from any Postgres host and to restore that dump.
+
+### Restoring from an RDS snapshot
+
+#### 1. Identify the snapshot you want to restore
+
+```
+$ aws rds describe-db-snapshots --db-instance-identifier elife-xpub-staging --no-paginate | jq -r .DBSnapshots[].DBSnapshotIdentifier
+rds:elife-xpub-staging-2019-01-26-03-30
+rds:elife-xpub-staging-2019-01-27-03-30
+rds:elife-xpub-staging-2019-01-28-03-30
+rds:elife-xpub-staging-2019-01-29-03-30
+```
+
+The format of the date in the snapshot name is `YYYY-mm-dd-HH-MM`.
+
+#### 2. Get the subnet group and security group of the existing database instance
+
+```
+$ aws rds describe-db-instances --db-instance-identifier=elife-xpub-staging | jq .DBInstances[0]
+[
+    "DBSubnetGroup": {
+        "DBSubnetGroupName": "elife-xpub-staging-attacheddbsubnet-htu8tybjp0f4",
+        ...
+    },
+    "VpcSecurityGroups": {
+        "VpcSecurityGroupId": "sg-013a86123118c6cc8",
+        "Status": "active"
+    },
+    ...
+]
+```
+
+The RDS instance that will be created from the snapshot has to use the same subnet and security group, so that it is accessible from the `elife-xpub` servers of that environment.
+
+#### 3. Create a new RDS instance from the snapshot, in the correct subnet group
+
+```
+$ aws rds restore-db-instance-from-db-snapshot --db-instance-identifier elife-xpub-staging-restore --db-snapshot-identifier rds:elife-xpub-staging-2019-01-29-03-30 --db-instance-class db.t2.small  --db-subnet-group-name elife-xpub-staging-attacheddbsubnet-htu8tybjp0f4
+```
+
+Wait for the instance to become available:
+
+```
+$ aws rds describe-db-instances --db-instance-identifier elife-xpub-staging | jq -r .DBInstances[0].DBInstanceStatus
+available
+```
+
+#### 4. Modify the instance with the correct security group
+
+```
+$ aws rds modify-db-instance --db-instance-identifier elife-xpub-staging-restore --vpc-security-group-ids sg-013a86123118c6cc8
+{
+	"DBInstance": {
+        "Endpoint": {
+            "Port": 5432,
+            "Address": "elife-xpub-staging-restore.cxyopn44uqbl.us-east-1.rds.amazonaws.com",
+            "HostedZoneId": "Z2R2ITUGPM61AM"
+        },
+		...
+}
+```
+
+#### 5. Dump the new node
+
+As the prompt shows, you have to do this from node 1 of an `elife-xpub` cluster, which is allowed to access the database nodes and has the necessary scripts deployed.
+
+```
+elife@staging--xpub:~$ dump-database.sh /ext/tmp/dump-afternoon elife-xpub-staging-restore.cxyopn44uqbl.us-east-1.rds.amazonaws.com
+```
+
+The `PGHOST` being used here is the RDS node that has just been created (2nd argument).
+
+#### 6. Restore the dump on the existing node
+
+```
+ ./restore-database.sh /ext/tmp/dump-afternoon elife-xpub-staging.cxyopn44uqbl.us-east-1.rds.amazonaws.com
+```
+
+The `PGHOST` being used here is the existing RDS node for this environment (2nd argument).
```

Comment:
Do we need instructions for removing the "new node" ... which I guess was only done to perform the SQL extraction?

Reply:
Assuming this is done in an emergency, there's no harm in leaving it around, an administrator can remove it later on; didn't want to make this guide even longer.